libs/MeshKernel/include/MeshKernel/Constants.hpp

```diff
@@ -67,6 +67,7 @@ namespace meshkernel
             constexpr double earth_radius = 6378137.0;                  ///< Earth radius(m)
             constexpr double inverse_earth_radius = 1.0 / earth_radius; ///< One over constants::geometric::earth_radius(m-1);
             constexpr double absLatitudeAtPoles = 0.0001;               ///< Pole tolerance in degrees
+            constexpr double refinementTolerance = 1.0e-2;              ///< Relative size of refinement.
```

Comment:
Before addressing this comment, you used `double tolerance = 1.0e-3;`. This became here 1.0e-2. What happened?

Reply:
When refining the polygons, each polyline segment is refined so that it has refined segments of x-metres. The tolerance is used to control the size of the smallest refined segment.

Example, the original segment length is 1001 metres, the user wants to have a maximum segment length of 100 metres, the algorithm will divide the original segment into 10 segments of 100 metres and 1 of 1 metre. This is the limit of the tolerance. If the original lenght was 1000.9 then there would be 9 segments of 100 metres and 1 of 100.9 metres.

I have two issues with the algorithm as it currently stands:
1. I think that 1.0e-2 is also too big, probably 0.1 might be a better choice
2. better still would be to have an option for a uniform refinement, but this is different from the original algorithm or would require an API change (an extra parameter indicating a uniform spacing is required)